docs/UserGuide.md

```diff
@@ -253,29 +255,35 @@ Example:
 
 ### Reschedules an appointment for a patient : `change`
 
-Reschedules or modifies an existing appointment with a new date, time and duration.
+Reschedules or modifies an existing appointment with a new date, time or duration.
 
-Format: `change APPT_INDEX d/DATE t/TIME dur/DURATION`
+Do note that the duration used here is **measured in minutes**!
+
+Format: `change APPT_INDEX [d/DATE] [t/TIME] [dur/DURATION]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-Do note that the duration used here is measured in minutes!
+If you would only like to change the duration of an appointment while keeping it's original date and time, simply 
+enter the new `DURATION` with it's corresponding `APPT_INDEX`! There is no need to retype it's date and time. The same 
+applies for the other fields.
 </div>
 
-* Reschedules an appointment at the specified `APPT_INDEX` for the patient it is tagged to. 
-The index here refers to the index number shown on the left side of the displayed appointment list on the right side of the UI. 
+* **Reschedules an appointment** at the specified `APPT_INDEX` for the patient it is tagged to. 
+* The `APPT_INDEX` here refers to the number shown on the left side of the displayed appointment book located on the right side of the UI. 
```

Comment:
It feels abit long-winded to me. I think can remove the `left side` part?  
Lemme know what you think about it? :smile: